src/layouts/partials/menu.html

```diff
@@ -3,27 +3,28 @@
 <section class="nav-wrap">
 	<nav class="acnav" role="navigation">
 		<ul class="acnav__list acnav__list--level1">
+
 			{{ $currentPage := . }}
             {{ range .Site.Menus.main }}
                 {{ $pageForItem1 := $site.GetPage .URL }}
                 {{ if .HasChildren }}
                     <li class="has-children">
-                        <div class="acnav__label{{ if or ($currentPage.HasMenuCurrent "main" .) ($currentPage.IsMenuCurrent "main" . ) }} active{{ end }}"><a href="{{ .URL }}" class="{{ if $currentPage.IsMenuCurrent "main" . }}current{{ end }}">{{ .Name }}</a></div>
-                        {{ if $currentPage.IsDescendant $pageForItem1 }}
+                        <div class="acnav__label{{ if or ($currentPage.HasMenuCurrent "main" .) (eq $pageForItem1.Name $currentPage.Name) }} active{{ end }}"><a href="{{ .URL }}" class="{{ if $currentPage.IsMenuCurrent "main" . }}current{{ end }}">{{ .Name }}</a></div>
+                        {{ if or ($currentPage.IsDescendant $pageForItem1) (eq $pageForItem1.Name $currentPage.Name) }}
```

Comment:
For some weird reason the function `IsMenuCurrent` does not return right value, so I had to fallback to compare the Name (ID is no available).

Reply:
Strange indeed. Did you try [HasMenuCurrent](https://gohugo.io/methods/page/hasmenucurrent/) as well?

Reply:
yes does not work, returns false 🤷